mitmproxy-rs/src/server/local_redirector.rs

```diff
@@ -62,6 +65,25 @@ impl LocalRedirector {
         self.server.wait_closed(py)
     }
 
+    /// Returns a `str` describing why local redirect mode is unavailable, or `None` if it is available.
+    ///
+    /// Reasons for unavailability may be an unsupported platform, or missing privileges.
+    #[staticmethod]
+    pub fn unavailable_reason() -> Option<String> {
+        #[cfg(any(windows, target_os = "macos"))]
+        return None;
+
+        #[cfg(target_os = "linux")]
+        if !unistd::geteuid().is_root() {
+            Some(String::from("mitmproxy is not running as root"))
+        } else {
+            None
+        }
```

Comment:
Can we comment this part out until Linux actually works?

(I want to cut a release right away to unblock @lups2000 on the front end)